src/rpc/blockchain.cpp

```diff
@@ -30,8 +30,6 @@
 
 #include <univalue.h>
 
-#include <boost/thread/thread.hpp> // boost::thread::interrupt
```

Comment:
Disagree:
https://github.com/bitcoin/bitcoin/blob/1a30cde4e7c0f2d78ce5b515d013429961122126/src/rpc/blockchain.cpp#L830

Which uses this header. See http://www.boost.org/doc/libs/1_54_0/doc/html/thread/thread_management.html.